components/ShadowLayer/examples/ShadowCornerRadiusExample.m

```diff
@@ -104,6 +105,19 @@ - (void)viewDidLoad {
   [self.view addSubview:_shadowsView];
 }
 
+- (void)viewDidLayoutSubviews {
```

Comment:
Just curious, would it be good or not to make the changes in `viewWillLayoutSubviews` since it could affect how its view layout subviews (which might include shadowsView)? What would be the concerns? Thanks!

Reply:
In this case we're enforcing a specific manual layout, so we want to have the "final say" in the subview frames. I think calling `viewDidLayoutSubviews` is preferred in this regard.